src/webui/www/private/newcategory.html

```diff
@@ -66,6 +117,16 @@
                     return true;
                 };
 
+                const body = {};
```

Comment:
Not sure if this is worth it but you can use a `Map` instead of a `Object` as such:
```js
const body = new Map();
body.set("downloadPathEnabled", false);

const params = {
    ...Object.fromEntries(body),
}
```